sum_from_string.py

```diff
@@ -0,0 +1,9 @@
+"""Sum from string."""
+
+
+import re
+
+
+def sum_from_string(string):
+    """Return sum of all numbers in a string."""
+    return sum(map(int, re.findall('\d+', string)))
```

Comment:
regex might be more than needed here; you could also use `filter` and `str.isdigit`:

`return sum(map(int, filter(str.isdigit, string))))`

This is kind of interesting, because it implies you are passing an instance of str into the `isdigit` method of the `str` type itself. Try it out!